01-node-tutorial/10-fs-sync.js

```diff
@@ -1,3 +1,5 @@
+
+
 const { readFileSync, writeFileSync } = require('fs')
 console.log('start')
 const first = readFileSync('./content/first.txt', 'utf8')
```

Comment:
Im not really sure about where you get the instruction from but in the assignement I saw that they said to create a file from `./temporary/fileA.txt` and add it to `.gitignore` file that that you don't have to push it on gitHub